app/src/renderer/components/staking/PageBond.vue

```diff
@@ -66,65 +70,59 @@ export default {
     ToolBar
   },
   computed: {
-    ...mapGetters(['shoppingCart', 'user']),
-    reservedAtoms () {
-      return this.shoppingCart.reduce((sum, d) => sum + (d.reservedAtoms || 0), 0)
+    ...mapGetters(['shoppingCart', 'user', 'committedDelegations']),
+    previouslyBondedAtoms () {
+      return Object.values(this.committedDelegations).reduce((sum, d) => sum + d, 0)
     },
-    unreservedAtoms () {
-      return this.user.atoms - this.reservedAtoms
+    willUnbondAtoms () {
+      let sum = 0
+      for (let id of Object.keys(this.committedDelegations)) {
+        let committed = this.committedDelegations[id]
+        let delegate = this.fields.delegates.find(c => c.id === id)
+        let willBond = delegate ? delegate.atoms : 0
+        let unbondAmount = Math.max(committed - willBond, 0)
+        sum += unbondAmount
+      }
+      return sum
     },
     unbondedAtoms () {
-      let value = this.unreservedAtoms
-
-      // reduce unreserved atoms by bonded atoms
-      this.fields.delegates.forEach(f => (value -= f.atoms))
-
-      return value
+      let willBondSum = this.bondedAtoms
+      let bondedSum = this.previouslyBondedAtoms
+      return this.user.atoms - willBondSum + bondedSum - this.willUnbondAtoms
     },
     unbondedAtomsPct () {
-      return Math.round(this.unbondedAtoms / this.user.atoms * 100 * 100) / 100 + '%'
+      return Math.round(this.unbondedAtoms / this.totalAtoms * 100 * 10) / 10 + '%'
     },
     bondedAtoms () {
-      let value = 0
-      this.fields.delegates.forEach(f => (value += f.atoms))
-      return value
+      return this.fields.delegates.reduce((sum, d) => sum + (d.atoms || 0), 0)
     },
     bondedAtomsPct () {
-      return Math.round(this.bondedAtoms / this.user.atoms * 100 * 100) / 100 + '%'
+      return Math.round(this.bondedAtoms / this.totalAtoms * 100 * 10) / 10 + '%'
+    },
+    totalAtoms () {
+      return this.bondedAtoms + this.unbondedAtoms
     }
   },
   data: () => ({
     equalize: false,
-    atomsMin: 1,
-    reservedAtomsMin: 0,
+    atomsMin: 0,
     fields: {
-      reservedAtoms: 0,
       delegates: []
     }
   }),
   methods: {
     fillAtoms (delegateId) {
```

Comment:
what is `fillAtoms`?

Reply:
I'm not really sure, but it's not being used anymore

Reply:
`fillAtoms` was used in the Equalize button to automatically dump all your atoms to the delegation. It was mainly for testing. We don't need it anymore.